ArcadiaParties-SPA/src/app/user/user-info/user-info.component.scss

```diff
@@ -0,0 +1,15 @@
+:host {
+  display: flex;
+  flex: 1 0 auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.center {
+  text-align: center;
+  font-size: 15pt;
```

Comment:
This doesn't correlate with class name, that's why this is confusing.

Reply:
Class added

Reply:
This is not what I meant indeed, but ok, let it be.